.github/workflows/docker-nightly-publish.yml

```diff
@@ -70,29 +79,25 @@ jobs:
         run: |
           ./gradlew --refresh-dependencies :serving:dockerDeb -Psnapshot
       - name: Build and push nightly docker image
-        if: ${{ inputs.mode == '' || inputs.mode == 'nightly' }}
         working-directory: serving/docker
         run: |
           export NIGHTLY="-nightly"
           docker compose build --no-cache \
           --build-arg djl_version=${{ env.DJL_VERSION }}-SNAPSHOT \
           --build-arg djl_serving_version=${{ env.SERVING_VERSION }}-SNAPSHOT \
           ${{ matrix.arch }}
-          docker compose push ${{ matrix.arch }}
-      - name: Build and push temp image
-        if: ${{ inputs.mode == 'temp' }}
+      - name: Tag and push temp image to ECR repo
+        if: ${{ !inputs.skip_nightly_integ_test && inputs.mode == 'nightly' ||  inputs.mode == 'temp'}}
         working-directory: serving/docker
         run: |
-          export NIGHTLY="-nightly"
-          docker compose build --no-cache \
-          --build-arg djl_version=${{ env.DJL_VERSION }}-SNAPSHOT \
-          --build-arg djl_serving_version=${{ env.SERVING_VERSION }}-SNAPSHOT \
-          ${{ matrix.arch }}
-          repo="185921645874.dkr.ecr.us-east-1.amazonaws.com/djl-ci-temp"
-          aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin $repo
-          tempTag="$repo:${{ matrix.arch }}-${GITHUB_SHA}"
-          docker tag deepjavalibrary/djl-serving:${{ matrix.arch }}-nightly $tempTag
+          aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin ${{env.AWS_ECR_REPO}}
+          tempTag="${{ env.AWS_ECR_REPO }}:${{ matrix.arch }}-${GITHUB_SHA}"
```

Comment:
This will result in new temp images in ECR nightly. Have we configured the retention policy in this ECR repo? We should discuss what that retention period looks like

Reply:
Someone created the ECR repo already with 7-day retention policy.

Reply:
The price for ECR storage is $0.09 per GB per month. ~$90 per TB per month.
If we want to make it more frugal. I believe a retention policy of 2days is enough since tests only last a few hours

Reply:
I am not really worried about the cost here. I am worried about the time wasted in saving and retrieving images from ECR. ECR is a fairly slow service.